app/styles/main.css

```diff
@@ -814,3 +810,60 @@ table.districts p {
     background-image: url(../images/passrate_student_yellow.png);
     background-repeat: no-repeat;
 }
+
+
+/*header*/
+.navbar-brand.header-flag{
+  float:left;
+  margin-right:10px;
+  padding-top:10px;
+  padding-left:0;
+}
+
+/*header - navigation*/
+.navbar-collapse{
+  float:right;
+}
+
+/*language nav*/
+.navbar-collapse.navbar-lang{
+  float:right;
+  background: #f0f0f0;
+  margin-top:10px;
+  padding-top:17px;
+  padding-bottom:13px;
+  -webkit-border-radius: 5px;
```

Comment:
we have an autoprefixer task in the build, so you can just do `border-radius: 5px;` and the rest will be added automatically